app/charts/shared/legend-color.tsx

```diff
@@ -309,7 +315,7 @@ const LegendColorContent = (props: LegendColorContentProps) => {
     }
   });
 
-  return (
+  return visible ? (
```

Comment:
@noahonyejese I think this variable should only show / hide legend title, not the whole legend – did I get this wrong? 🤔